HomeWorks/HW05.Task3/Reversearraymanual.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HW05.Task3
+{
+    public class Reversebymyself
+    {
+        public static void Reversearraymanual()
+        {
+            int[] arraybeforereverse = new int[] { 1, 2, 3, 4, 5, 6, 7 };
+            foreach (var i in arraybeforereverse)
+            {
+                Console.WriteLine($"reversearray[{i - 1}]:{arraybeforereverse[i - 1]}");
+            }
+            
+            int[] arrayafterreverse = new int[7];
```

Comment:
what about if i would like to reverse another array with size 1_000? Your method works only with array[7]